### ISSUE TITLE
ISO 690 ES

### ISSUE THREAD

**Issue by bwiernik** [citation-style-language/styles#7304]
Hello! Thank you for all the changes. I was checking it and I found some issues.
Also I found a pdf of an Spanish university explaining how the iso 690:2024 works (newest version)
I took it everything in account and made a pdf linked in this message with all the modifications highlighted in red (both from the issues as well as the ones from the new version). I was wondering if you could please make them.
I was also wondering if we could get the cite style ready to cite also with ibid, idem and op.cit (it is also explained in the word file).
I am also uploading the presentation of the University of Sevilla explaining everything of the new version (may be useful)
in any case I am available to answer any doubts

Thank you very much again

[Modifications.pdf](https://github.com/user-attachments/files/17611185/Modifications.pdf)
[citas_y_referencias_eps_2024_1.pdf](https://github.com/user-attachments/files/17611189/citas_y_referencias_eps_2024_1.pdf)

Could you take a look @POBrien333 ?

_Originally posted by @anamulopez in https://github.com/citation-style-language/styles/pull/7061#issuecomment-2453488431_

**Comment by POBrien333** [comment]
Is this about the spanish note style?
Am I interpreting it correctly that the stuff in red is missing?

**Comment by anamulopez** [comment]
Yes. As far as I've understood in the pdf of the University of Sevilla. The data in red in missing. 
There are some errors also in red which appear crossed
       - In the articles, the issue appears as "n.º." and should be "nº" without the points
       - In the webpage, before the link it says "En. Disponible en:" which is redundant. It should be only "Disponible en:"
 
Latter, in the pages where I explain the ibid-idem-opcit. I have used the red to evidence where they appear and how they work.

**Comment by POBrien333** [comment]
@anamulopez 
Did you have a chance to test out this style?
https://github.com/citation-style-language/styles/raw/3cf870f072dcec2ff6f3febcfea043b29d7073be/iso690-full-note-es.csl

**Comment by anamulopez** [comment]
Hello again. 

I've been comparing both the citation in footnote and bibliography and there is some little data missing. I'm showing it in red in the pdf attached. I also added the instuctions to make the Ibid- idem- opcit work when the cites repeat in the text. It'd be great to add it too.

Thank you very much fot all the effort

[Modifications 202501.pdf](https://github.com/user-attachments/files/18302196/Modifications.202501.pdf)
